web/styles/compose.css

```diff
@@ -266,8 +266,46 @@
 #message-content-container {
     grid-area: message-content-container;
     display: grid;
-    grid-template: minmax(0, 1fr) / minmax(0, 1fr);
+    grid-template: minmax(0, 1fr) / minmax(0, 1fr) 25px;
     margin-top: 5px;
+
+    &:hover {
+        .composebox-buttons > button {
+            opacity: 0.7;
+            visibility: visible;
+        }
+    }
+}
+
+#message-content-container .composebox-buttons {
+    z-index: 1;
```

Comment:
This z index is needed as otherwise hovering when previewing causes weird flickering of the button and the cursor.

Reply:
Let's be sure to document that in the code here.

Reply:
Done!